.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: java
 jdk:
   - oraclejdk8
-sudo: required
 
-jobs:
-  include:
-    - stage: build
-      script: mvn clean package
-      if: type = push AND branch != master
```

Comment:
Why these branch tests are removed? We do not need to do different builds for different branches and webhook types?

Reply:
the new script has covered that already.